react/getting_started_with_react/what_is_jsx.md

```diff
@@ -212,9 +212,11 @@ Now that we've applied all of the fixes to the errors that React gave us, this i
 
 <div class="lesson-content__panel" markdown="1">
 
-1.  Read through the React docs [Writing markup with JSX page](https://react.dev/learn/writing-markup-with-jsx) to review the content we have covered here, and complete the challenge at the end.
+1.  In the following lessons, you will spend some time reading the React documentation, such as the articles below. Most of them will have small tasks at the end to test what you have read. We will not mention them explicitly each time, but be sure to do them. Practice makes perfect!
```

Comment:
I don't think this suggestion needs to be part of the list

Reply:
I think it would make sense, as it emphasises the importance of the tasks.  Ultimately, it is an assignment. Anyway, I can also incorporate your comment. Should I do another pull request and add the information on top of the assignment block?

Reply:
I think it should part of the assignment block, just not in the list.

No need for another pull request, just push another commit to this current branch. It will be added in this PR automatically